### PR TITLE
Ignore internal eventbus metrics

### PR DESCRIPTION
**Pull request by jotak**
See issue #39

### CODE REVIEW

**Comment by jotak** [comment]
@vietj / @tsegismont if it's ok with you I'll do a similar patch in dropwizard module

Ps: do you think there's any need in adding a config flag to turn on/off ignoring internal EB addresses?

**Comment by vietj** [comment]
@jotak that would be great

File: src/main/java/io/vertx/micrometer/impl/VertxEventBusMetrics.java [hunk 1]
```diff
@@ -58,15 +58,25 @@
...
   }
 
+  private static boolean isInternal(String address) {
+    return address.startsWith("__vertx.");
+  }
+
   @Override
   public Handler handlerRegistered(String address, String repliedAddress) {
+    if (isInternal(address)) {
+      // Ignore interna metrics
```
Comment (tsegismont) [inline src/main/java/io/vertx/micrometer/impl/VertxEventBusMetrics.java]: Typo _interna**l**_

File: src/main/java/io/vertx/micrometer/impl/VertxEventBusMetrics.java [hunk 2]
```diff
@@ -137,5 +161,13 @@ public void close() {
     Handler(String address) {
       this.address = address;
     }
+
+    static Handler ignored() {
+      return new Handler(null);
```
Comment (tsegismont) [inline src/main/java/io/vertx/micrometer/impl/VertxEventBusMetrics.java]: How about returning a constant instead of creating a new instance?

**Comment by jotak** [comment]
@tsegismont done

**Comment by tsegismont** [comment]
Thanks @jotak !

**Pull request merged**